dependency-graph/properties/dependency-review.properties.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "Dependency Review",
+    "creator": "GitHub",
+    "description": "Scans Pull Requests on each push for the introduction and/or resolution of vulnerable depdendencies to the repository",
+    "iconName": "octicon mark-github",
+    "categories": [
+      "Dependency Graph",
+      "Dependency Review",
+      "Go",
+      "Java",
+      "JavaScript",
+      "TypeScript",
+      "Python",
+      "Ruby",
+      "Actions",
+      "PHP"]
+}   
```

Comment:
I added tags that are not specific to language ecosystems, since our Action again is not a 1:1 match with `code-scanning` as described in PR template. Do I need to update this?